src/game/people.c

```diff
@@ -2,18 +2,15 @@
 
 #include "game/box.h"
 #include "game/control.h"
-#include "game/effects/blood.h"
-#include "game/effects/ricochet.h"
+#include "game/effects/gunshot.h"
 #include "game/items.h"
 #include "game/random.h"
-#include "game/sound.h"
-#include "game/sphere.h"
 #include "global/vars.h"
 
-int32_t Targetable(ITEM_INFO *item, AI_INFO *info)
+bool Targetable(ITEM_INFO *item, AI_INFO *info)
```

Comment:
`IsTargetable` or `IsLaraTargetable`?

Reply:
I didn't move it yet, but probably will want to move it to gunshot.c later. For now I want to keep it here.